offixsdk/src/main/java/org/aerogear/offixsdk/Application.kt

```diff
@@ -0,0 +1,40 @@
+package org.aerogear.offixsdk
+
+import android.app.Application
+import android.arch.persistence.room.Room
+import android.content.Context
+import android.net.ConnectivityManager
+import android.util.Log
+import org.aerogear.offixsdk.persistence.Database
+
+class Application : Application() {
+
+    companion object {
+        lateinit var database: Database
+    }
+
+    override fun onCreate() {
+        super.onCreate()
+        database = Room.databaseBuilder(this, Database::class.java, "libb_db.db")
```

Comment:
I think if we need context we should be requiring people to initialize this by function inside their application rather than have it here.

@harshithdwivedi not sure about this but it feels strange. Can I have your opinion?

Reply:
Yeah, you are right. We can't keep an application class in the library as if our client will use it he/she will not be able to make an application class of its own as an app can only have 1 application class.

Reply:
+1; let's get rid of the Application class and use a utils class/object instead